src/openklant/components/klantinteracties/api/filterset/digitaal_adres.py

```diff
@@ -131,6 +131,7 @@ class Meta:
             "is_standaard_adres",
             "omschrijving",
             "referentie",
+            "verificatie_datum",
```

Comment:
This means we can only filter on exact verification dates, which I don't think is that useful. I think we should at least add `gt`, `lt`, `gte` and `lte` to make it a bit more useful.

I also think it would be good to have a read only field `geverifieerd` + a filter for that